src/Foundation/Console/InfoCommand.php

```diff
@@ -91,20 +116,20 @@ protected function fire()
      * @param string $fallback
      * @return string
      */
-    private function findPackageVersion($path, $fallback)
+    private function findPackageVersion($path, $fallback = null)
     {
         if (file_exists("$path/.git")) {
             $cwd = getcwd();
             chdir($path);
 
             $output = [];
             $status = null;
-            exec('git rev-parse HEAD', $output, $status);
+            exec('git rev-parse HEAD 2> /dev/null', $output, $status);
```

Comment:
Hmm, how does this behave on Windows?

Reply:
@datitisev See commit 96045ca, that should fix it, I think.

Reply:
@franzliedke Looks like it still works on macOS. :+1: